mne/bem.py

```diff
@@ -1687,11 +1689,17 @@ def _prepare_env(subject, subjects_dir):
     return env, mri_dir, bem_dir
 
 
+@deprecated('convert_flash_mris is deprecated and will be removed in 1.2. '
+            'To convert your dicom files to .nii or .mgz files, use dcm2niix '
+            'or mri_convert available in FreeSurfer.')
 @verbose
 def convert_flash_mris(subject, flash30=True, convert=True, unwarp=False,
                        subjects_dir=None, verbose=None):
     """Convert DICOM files for use with make_flash_bem.
 
+    DEPRECATED: To convert your dicom files to .nii or .mgz files, use dcm2niix
+    or mri_convert available in FreeSurfer.
+
```

Comment:
No need to add this, it gets added automatically by the decorator, see how there are two warnings here now (and this second one is not as good)

https://output.circle-artifacts.com/output/job/eea3d6ff-add1-4169-a880-09860a695e4d/artifacts/0/dev/generated/mne.bem.convert_flash_mris.html

Also the function should be removed from `forward.rst`, and make sure any refs to it in `doc/changes/*` get replaced with double-backticks because `:func:` target will disappear